lib/request-execution.js

```diff
@@ -102,49 +103,50 @@ class RequestExecution {
   }
 
   _sendOnConnection() {
-    const self = this;
-    const startTime = process.hrtime();
+    this._startTime = process.hrtime();
```

Comment:
I didn't find enough evidence that this refactor (use an instance member instead of a closure) has an impact but in any case, having `this` context is less error prone.